bin/file_extension.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import sys
+
+from reusable import json_dump, package_validation_dict
+
+
+def file_extension_test(file_path: str):
+    """
+    1. Reads file extension from provided path
+    2. Checks for valid file extension (.txt is used for now, future version will use ome.tiff and ome.tif)
+    3. Returns result of File extension test ("pass" or "fail")
```

Comment:
Nit: I like the steps you put here, but also the 3rd point is repetitive with the `Returns:`.  Personally, I think what you have on the 2nd bullet point is good for this function description.